p2p/transport/webtransport/transport_test.go

```diff
@@ -584,3 +592,192 @@ func TestSNIIsSent(t *testing.T) {
 	}
 
 }
+
+func TestServerSendsBackValidCert(t *testing.T) {
+	var maxTimeoutErrors = 10
+
+	require.NoError(t, quick.Check(func(timeSinceUnixEpoch time.Duration, keySeed int64, randomClientSkew time.Duration) bool {
```

Comment:
Do we need to repeat this test 100 time or so?

Reply:
That's the default: https://pkg.go.dev/testing/quick?utm_source=gopls#Config